frontend/static/js/index.js

```diff
@@ -0,0 +1,57 @@
+import Design from "./views/Design.js";
+import Tech from "./views/Tech.js";
+import Apply from "./views/Apply.js";
+import NotFound from "./views/NotFound.js";
+
+const router = () => {
+  const routes = [
+    {
+      path: "/",
+      view: Tech,
+    },
+    {
+      path: "/design",
+      view: Design,
+    },
+    {
+      path: "/tech",
+      view: Tech,
+    },
+    {
+      path: "/apply",
+      view: Apply,
+    },
+    {
+      path: "/404",
+      view: NotFound,
+    },
+  ];
+  const Matches = routes.map((route) => {
+    return {
+      route,
+      isMatch: route.path === location.pathname,
+    };
+  });
+
+  let match = Matches.find((Match) => Match.isMatch);
```

Comment:
p0
마이너한 코멘트이지만 카멜케이스로 작성 하는게 좋아보입니다. 일반적으로 파스칼 케이스는 클래스를 나타내는 룰이기 때문입니다.

```suggestion
  const matches = routes.map((route) => {
    return {
      route,
      isMatch: route.path === location.pathname,
    };
  });

  let match = matches.find((Match) => match.isMatch);
```